Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM node:6.17.0-stretch
+FROM node:17.3-bullseye-slim
```

Comment:
:raised_hands: were you able to build the image locally and test it via the CLI?

Reply:
yes! CLI returned no errors and a lot less vulnerabilities

Reply:
Awesome